python/paddle/nn/functional/loss.py

```diff
@@ -3884,3 +3887,132 @@ def soft_margin_loss(input, label, reduction='mean', name=None):
         return paddle.mean(out, name=name)
     else:
         return out
+
+
+def gaussian_nll_loss(
+    input, target, var, full=False, eps=1e-6, reduction='mean', name=None
```

Comment:
In the [new API development&submission process](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_contributing_guides_cn.html), there are [API design and naming specifications](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_design_guidelines_standard_cn.html). `target`, `var` and `eps` are not recommended. It is recommended to change to `label`, `varible` and `epsilon` to keep consistent with other APIs

Reply:
this also needs to be modified in rfc

Reply:
Thanks, I realize.